ReactWindows/ReactNative/Views/TextInput/ReactTextInputShadowNode.cs

```diff
@@ -22,7 +21,16 @@ public class ReactTextInputShadowNode : LayoutShadowNode
     {
         private const int Unset = -1;
 
+        private static readonly float[] s_defaultPaddings = new[]
```

Comment:
where did we get these numbers from?  Just from the text input itself with default settings?